docker/README.md

```diff
@@ -0,0 +1,80 @@
+# Quick Start
```

Comment:
Extend the top-level directory's README: https://github.com/kmesh-net/orion?tab=readme-ov-file#quick-start

Reply:
Maybe you could keep this detailed README and extend the top-level README with minimal operations, including only build, run and verify the Docker image.